gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/ArtifactService.java

```diff
@@ -66,16 +66,14 @@ public ArtifactService(
 
   public List<Map> getArtifactCredentials(String selectorKey) {
     return mapListCommand(
-            "artifactCredentials",
-            clouddriverServiceSelector.select(selectorKey)::getArtifactCredentials)
+            "artifactCredentials", clouddriverServiceSelector.select()::getArtifactCredentials)
```

Comment:
`selectorKey` is no longer used (prob could go in a separate PR to clean those up..) (same in a few below)

Reply:
yes this PR was already on the big side, might do an opportunistic cleanup later